timeinterval/timeinterval_test.go

```diff
@@ -659,3 +660,97 @@ func mustLoadLocation(name string) *time.Location {
 	}
 	return loc
 }
+
+func TestIntervener_Mutes(t *testing.T) {
```

Comment:
These tests are technically a duplicate of what we already have in `notify` - I think they have some merit in being here as there might be more we want to test as part of this interface in the future (e.g. when we introduce the marker) and it gave me more confidence in the refactor.